armometadata/armometadata.go

```diff
@@ -22,6 +22,9 @@ type ClusterConfig struct {
 	GatewayRestURL            string `json:"gatewayRestURL"`            // in-cluster gateway component REST API url
 	KubevulnURL               string `json:"kubevulnURL"`               // in-cluster kubevuln component REST API url
 	KubescapeURL              string `json:"kubescapeURL"`              // in-cluster kubescape component REST API url
+	StorageEnabled            bool   `json:"storage"`                   // storage configuration (enabled/disabled)
+	NodeAgentEnabled          bool   `json:"nodeAgent"`                 // node agent configuration (enabled/disabled)
```

Comment:
let's add all the features.
image scan
posture scan
otel collector

Reply:
@Daniel-GrunbergerCA